controller/controlplane/controller_reconciler_utils.go

```diff
@@ -671,15 +766,42 @@ func (r *Reconciler) ensureValidatingWebhookConfiguration(
 ) (op.Result, error) {
 	logger := log.GetLogger(ctx, "controlplane.ensureValidatingWebhookConfiguration", r.DevelopmentMode)
 
-	managedByLabelSet := k8sutils.GetManagedByLabelSet(cp)
+	// TODO: This performs a migration from the old managedBy label to the new one.
```

Comment:
Is the migration performed here? Is the new label set enforced or the migration is something we still need to address?

Reply:
I see. Could you rephrase the comment to clearly state this aspect? It's not immediate to me that the TODO is related to removing the migration logic. Also, mentioning the #401 in the code would be great. WDYT?

Reply:
423c792f64b0fd4ff9a6c273fe2a42614a9aacca PTAL

Reply:
that's great 👍